src/cf-tables/usage.md

```diff
@@ -119,8 +119,8 @@ This striping is not visible on small screens.
 
 ## Right-aligned cells
 
-The use of the `.o-table_cell__right-align` class on a TD aligns the text right -
-see the third column above.
+The use of the `.o-table_cell__right-align` class on a `td` aligns the text right
+- see the third column above.
```

Comment:
Putting the hyphen at the start of a new line causes it to render as an unordered list item.